integration/data/test_backup.py

```diff
@@ -179,23 +180,70 @@ def test_backup_hole_with_backing_file(backing_dev):  # NOQA
     assert c == snap2_checksum
 
 
-def test_snapshot_tree_backup(dev):  # NOQA
+def snapshot_tree_backup(dev, backupTarget):  # NOQA
     offset = 0
     length = 128
     backup = {}
 
     snap, data = snapshot_tree_build(dev, offset, length)
 
-    backup["0b"] = cmd.backup_create(snap["0b"], BACKUP_DEST)
-    backup["0c"] = cmd.backup_create(snap["0c"], BACKUP_DEST)
-    backup["1c"] = cmd.backup_create(snap["1c"], BACKUP_DEST)
-    backup["2b"] = cmd.backup_create(snap["2b"], BACKUP_DEST)
-    backup["2c"] = cmd.backup_create(snap["2c"], BACKUP_DEST)
-    backup["3c"] = cmd.backup_create(snap["3c"], BACKUP_DEST)
+    backup["0b"] = cmd.backup_create(snap["0b"], backupTarget)
+    backup["0c"] = cmd.backup_create(snap["0c"], backupTarget)
+    backup["1c"] = cmd.backup_create(snap["1c"], backupTarget)
+    backup["2b"] = cmd.backup_create(snap["2b"], backupTarget)
+    backup["2c"] = cmd.backup_create(snap["2c"], backupTarget)
+    backup["3c"] = cmd.backup_create(snap["3c"], backupTarget)
 
     snapshot_tree_verify_backup_node(dev, offset, length, backup, data, "0b")
     snapshot_tree_verify_backup_node(dev, offset, length, backup, data, "0c")
     snapshot_tree_verify_backup_node(dev, offset, length, backup, data, "1c")
     snapshot_tree_verify_backup_node(dev, offset, length, backup, data, "2b")
     snapshot_tree_verify_backup_node(dev, offset, length, backup, data, "2c")
     snapshot_tree_verify_backup_node(dev, offset, length, backup, data, "3c")
+
+
+def test_backup(replica1, replica2, controller):  # NOQA
+    env = dict(os.environ)
+    assert env["BACKUPTARGETS"] != ""
+    dev = common.get_dev(replica1, replica2, controller)
+    if env["BACKUPTARGETS"] != "":
+        backupTargets = env["BACKUPTARGETS"].split(",")
+        for backupTarget in backupTargets:
+            backup(dev, backupTarget)
```

Comment:
`dev` will be reused by the next backupTarget. Need clean up as well.

Same for functions below.